streampark-common/src/test/scala/org/apache/streampark/common/fs/HdfsOperatorTest.scala

```diff
@@ -0,0 +1,300 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.fs
+
+import org.apache.streampark.common.fs.HdfsOperatorTest.withTempDir
+
+import org.apache.commons.codec.digest.DigestUtils
+import org.apache.commons.io.{FileUtils, IOUtils}
+import org.junit.jupiter.api.Assertions.{assertDoesNotThrow, assertEquals, assertFalse, assertNotEquals, assertThrows, assertTrue}
+import org.junit.jupiter.api.Test
+
+import java.io.{File, FileInputStream}
+import java.nio.file.{Files, Paths}
+
+object HdfsOperatorTest {
```

Comment:
There is currently no e2e test, so you can test whether your results are correct.